.github/workflows/ci.yaml

```diff
@@ -8,11 +8,44 @@ on:
   pull_request:
 
 jobs:
+  cs_fix:
+    name: Run code style check
+    runs-on: "ubuntu-20.04"
+    strategy:
+      matrix:
+        php:
+          - '8.0'
```

Comment:
If this step works with PHP 8.0, then the unit tests executed below should not `--ignore-platform-req php` anymore for PHP 8.0, possible 8.1 as well.